provisioning/packer-virtual-machines/packer-image/packer_builder/builder.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Be consistent. Since all other folders have dashes in their name, do not use underscores.